assets/js/modules/analytics/components/dashboard/DashboardPopularPagesWidget.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * DashboardPopularPagesWidget component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import whenActive from '../../../../util/when-active';
+import PreviewTable from '../../../../components/preview-table';
+import Layout from '../../../../components/layout/layout';
+import { getDataTableFromData, TableOverflowContainer } from '../../../../components/data-table';
+import { numberFormat } from '../../../../util';
+import getDataErrorComponent from '../../../../components/notifications/data-error';
+import getNoDataComponent from '../../../../components/notifications/nodata';
+
+const { useSelect } = Data;
+
+function DashboardPopularPagesWidget() {
+	const {
+		analytics,
+		data,
+		error,
+		loading,
+	} = useSelect( ( select ) => {
+		const store = select( STORE_NAME );
+		const args = {
+			dateRange: select( CORE_USER ).getDateRange(),
+			dimensions: 'ga:pageTitle,ga:pagePath',
+			metrics: [
+				{
+					expression: 'ga:pageviews',
+					alias: 'Pageviews',
+				},
+			],
+			orderby: [
+				{
+					fieldName: 'ga:pageviews',
+					sortOrder: 'DESCENDING',
+				},
+			],
+			limit: 10,
+		};
+
+		return {
+			analytics: select( CORE_MODULES ).getModule( 'analytics' ),
```

Comment:
Instead of the `analytics.homepage` below, we should rely on the `getServiceURL` selector of the `modules/analytics` store, similar like it is done in the Search Console Top Keywords widget in #1899. This way it works based on the datastore and does not rely on server-provided data (currently this link is not showing up as expected in Storybook because of that).